pi_piper-sysfs.gemspec

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'pi_piper/sysfs/version'
```

Comment:
Alright that sounds fair. I didn't want to offend you if I rewrote things a
bit. If you're ok with that I will merge the changes. I'm fine with the
rest, just some minor points really.

Still, thank you for contributing so far.
On Mar 21, 2016 2:30 AM, "Marco" <notifications@github.com> wrote:

> In pi_piper-sysfs.gemspec
> <https://github.com/PiPiper/sysfs/pull/1#discussion_r56787899>:
>
> > @@ -0,0 +1,32 @@
> > +# coding: utf-8
> > +lib = File.expand_path('../lib', __FILE__)
> > +$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
> > +require 'pi_piper/sysfs/version'
>
> Just let me finish, the implementation of the drivers I'm currently
> writing. Then you 'll be able to rewrite as you may wish.
>
> By the way the two last points of "Current approach" aren't true. current
> implementation allows to have methods or variables in the Driver superclass
> accessible by other drivers. they are just not accessible by bundler or
> rake (which doesn't need it anyway)
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/PiPiper/sysfs/pull/1/files/8e26d6f04c610c8c4e1d2af980cf8b8fba6c6533#r56787899>
>